### DISCUSSION TITLE
replace classes with JavaScript

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#8116] [Help] [answered]
I came across this [css tricks post](https://css-tricks.com/cycle-through-classes-html-element/#aa-cascading-replace-statements) about looping through multiple classes and I decided to try it myself, but it didn't work out: https://jsfiddle.net/xog8by5h/

**Comment by cannap** [comment, marked as answer]
you try to  replace classes on the event 

https://jsfiddle.net/9gzw32mx/ i added two ways one is from the el and one is from the e.target.
